src/gui/AutomationPatternView.cpp

```diff
@@ -278,6 +278,7 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 
 	const float y_scale = max - min;
 	const float h = ( height() - 2 * TCO_BORDER_WIDTH ) / y_scale;
+	const float ppTact  = ppt / MidiTime::ticksPerTact();
```

Comment:
This should actually be named `ppTick`, cause `ppt` already means pixels per tact, and you're dividing it with the number of ticks in a tact :slightly_smiling_face: